.circleci/config.yml

```diff
@@ -60,7 +60,32 @@ jobs:
 
       - run:
           name: Test Code
-          command: docker run --entrypoint "/app/.local/bin/coverage" fx-private-relay run --source=. -m pytest --cov=./ --cov-fail-under=60
+          command: |
+            # Create a volume owned by the app user
+            docker run \
+              --volume /tmp/test-results \
+              --name test-results \
+              alpine \
+              /bin/sh -c \
+                "chmod 0777 /tmp/test-results && \
+                 chown 10001:10001 /tmp/test-results"
+            # Run coverage tests, outputting the results in XML format
+            docker run \
+              --entrypoint "/bin/bash" \
+              --volumes-from test-results \
+              fx-private-relay \
+              -c \
+                'mkdir -p /tmp/test-results/pytest && \
+                 mkdir -p /tmp/test-results/coverage && \
+                 /app/.local/bin/coverage run --source=. --branch -m pytest \
+                   --cov=./ --cov-fail-under=60 \
```

Comment:
question: Now that this is done with multiple commands in a single `docker run` - will the `--cov-fail-under=60` still fail the step if the coverage drops under 60%?

Reply:
the command runs with `#!/bin/sh -eo pipefail`. `-e` means exit immediately when a command fails, and `-o pipefail` includes failures in the middle of a pipe operation. So, I think yes, it will fail if coverage falls below 60. But, there's only one way tko be 100% certain - remove some tests and watch it fail.